libopenage/coord/tile.h

```diff
@@ -1,11 +1,13 @@
-// Copyright 2016-2018 the openage authors. See copying.md for legal info.
+// Copyright 2016-2019 the openage authors. See copying.md for legal info.
 
 #pragma once
 
 #include "declarations.h"
 #include "coord_nese.gen.h"
 #include "coord_neseup.gen.h"
 #include "../util/misc.h"
+#include "../util/hash.h"
+#include <typeindex>
```

Comment:
when we introduce the helper function, we don't need the typeindex header anymore here.